README.md

```diff
@@ -267,8 +267,9 @@ fastify.register(require('fastify-swagger'), {
       querystring,
       headers,
       response,
+      ...restOfFastifySchema
```

Comment:
This create a new object already. Does it has any meaning to shallow clone later?
The original one is used to prevent modify of original object.

Reply:
No the goal of this PR was mainly to revert a last second commit of the last merged PR.

But I just looked at this example and at `joi-to-json` package after your review and in fact we can just destructure the original schema argument and parse it through `joi-to-json`.